package.json

```
@@ -7,6 +7,8 @@
   "type": "module",
   "scripts": {
     "dev": "pnpm run with-env vite dev",
+    "auth:login": "tsx scripts/auth-login.ts",
+    "with-env": "dotenv -e ../../.env",
```

Biome (2.4.12)

[error] The key with-env was already declared.

(lint/suspicious/noDuplicateObjectKeys)